scripts/weatherfx.js

```diff
@@ -251,9 +234,12 @@ async function checkWeather(msgString) {
 }
 
 // This function apply weather effects to the canvas, but first cleans any effects that are currently applied.
-function weatherEffects(effectCondition) {
-    clearEffects();
-    canvas.scene.setFlag("weatherfx", "active", true);
+async function weatherEffects(effectCondition) {
+    if (canvas.scene.getFlag("weatherfx", "isActive") !== undefined)
```

Comment:
Same as [here](https://github.com/LeafWulf/weatherfx/pull/1#discussion_r983997882) I am checking if the flag already exist so I know this scene had a weather effect applied at least once. Why though? 
I was getting an error when I run `function clearEffects()` because the effect condition flag would be undefined.
now it occurs to me though that if this is not a good practice and we have to remove it, we could change it to
 ```js
  if (canvas.scene.getFlag("weatherfx", "effectCondition")){
    ...
}
  ```

Reply:
Yup